### ISSUE TITLE
Homebrew always detects that fc4 is outdated even when it isn’t

### ISSUE THREAD

**Issue by aviflax** [FundingCircle/homebrew-floss#7]
```shell
$ brew outdated
fundingcircle/floss/fc4 (2019-07-26_1982) < 2019-07-26_1982
```

Probably related to #6 and https://github.com/FundingCircle/fc4-framework/issues/193

**Comment by aviflax** [comment]
@miketheman this is one of the issues I mentioned last night. Would love to get your 2¢ on this and #6 as well — thanks!

**Comment by aviflax** [comment]
This seems to have been resolved by #8:

```shell
$ brew list --versions | grep fc4
fc4 2019.09.10-c987903
$ brew outdated
$
```
